smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/PaginatorGenerator.kt

```diff
@@ -155,6 +154,25 @@ class PaginatorGenerator : KotlinIntegration {
                     }
                 }
             }
+
+        writer.write("")
+        writer
+            .dokka("""
+                $docBody
+                @param block A builder block used for DSL-style invocation of the operation
+                $docReturn
+            """.trimIndent())
+            .withBlock(
+                "fun #T.#LPaginated(block: #T.Builder.() -> Unit): #T<#T> =",
```

Comment:
`KotlinTypes.Unit` is available as well and we may want that to be fully qualified since there is a model with `Unit` as a shape IIRC